src/main/java/com/aws/greengrass/clientdevices/auth/metrics/ClientDeviceAuthMetrics.java

```diff
@@ -14,10 +14,12 @@
 import java.time.Instant;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.atomic.AtomicLong;
 import javax.inject.Inject;
 
 public class ClientDeviceAuthMetrics {
+    protected ScheduledFuture<?> future;
```

Comment:
why is this here?

Reply:
I added it to follow the format you sent earlier. I use it here: [https://github.com/aws-greengrass/aws-greengrass-client-device-auth/pull/204/files/4bd913d6d4157163b9b5e6a4eb5352cf6cf16061#diff-d3f582a458b026f2b5e5ec7d26a2671a2908c74f9d069e7930f2e90737795843R33](url)

Reply:
This doesn't belong here, this future has nothing to do with this class. It should just be in the MetricsEmitter